management/branding.go

```diff
@@ -168,21 +167,7 @@ func (m *BrandingManager) UniversalLogin(opts ...RequestOption) (ul *BrandingUni
 //
 // See: https://auth0.com/docs/api/management/v2#!/Branding/put_universal_login
 func (m *BrandingManager) SetUniversalLogin(ul *BrandingUniversalLogin, opts ...RequestOption) (err error) {
-	req, err := m.NewRequest("PUT", m.URI("branding", "templates", "universal-login"), ul.Body, opts...)
-	if err != nil {
-		return err
-	}
-
-	res, err := m.Do(req)
-	if err != nil {
-		return err
-	}
-
-	if res.StatusCode >= http.StatusBadRequest {
-		return newError(res.Body)
-	}
-
-	return nil
```

Comment:
All of this part was just replicating what the `m.Request()` was doing with the difference that we were not correctly closing the response body in this func, causing memory leaks.